app/controllers/holidays_controller.rb

```diff
@@ -80,19 +83,20 @@ def create_params
   end
 
   def batch_create_params
-    params
-      .require(:holiday)
-      .permit(
-        :title,
-        :all_day,
-        :start_at,
-        :end_at
-      )
-      .merge(users: user_ids)
+    params.require(:holiday)
+          .permit(
+            :title,
+            :all_day,
+            :single_day_start_at,
+            :single_day_end_at,
+            :multi_day_start_at,
+            :multi_day_end_at
+          )
+          .merge(users: user_ids)
   end
 
   def user_ids
-    params[:holiday][:users] || []
+    params[:holiday][:users].select(&:present?) || []
```

Comment:
This implies that that param is always an array right, empty or not. If so, `select` would return a `[]` anyway so the or clause is redundant.